tests/fixture/tmpnet/network.go

```diff
@@ -358,6 +362,48 @@ func (n *Network) Start(ctx context.Context, w io.Writer) error {
 	return nil
 }
 
+// Start the network for the first time
+func (n *Network) Bootstrap(ctx context.Context, w io.Writer) error {
+	if len(n.Subnets) == 0 || len(n.Nodes) == 1 {
+		// Start all nodes at once if a staged start is not worth it
+		return n.Start(ctx, w, n.Nodes...)
+	}
+
+	// Reduce the cost of subnet creation for a network of multiple nodes by
+	// creating subnets with a single node with sybil protection
+	// disabled. This allows the creation of initial subnet state without
+	// requiring coordination between multiple nodes.
+
+	bootstrapNode := n.Nodes[0]
+	if _, err := fmt.Fprintf(w, "Starting a single-node network with %s for quicker subnet creation\n", bootstrapNode.NodeID); err != nil {
+		return err
+	}
+	bootstrapNode.Flags[config.SybilProtectionEnabledKey] = false
+	if err := n.Start(ctx, w, bootstrapNode); err != nil {
+		return err
+	}
+	if err := n.CreateSubnets(ctx, w); err != nil {
+		return err
+	}
+
+	if _, err := fmt.Fprintf(w, "Restarting %s with sybil protection enabled to serve as bootstrap node\n", bootstrapNode.NodeID); err != nil {
+		return err
+	}
+	delete(bootstrapNode.Flags, config.SybilProtectionEnabledKey)
```

Comment:
I suppose we don't think it would make sense for the network to actually want sybil protection disabled in tests... I think that makes sense (it's really a flag intended for quick local testing... not for programatic testing